File: Algorithms/Source Code Algorithms/Sorting/SelectionSort/SCSelectionSort.c
void sort(int *a, int n) {
    for (int j = 0; j < n; j++) {
        int iMin = j;
        for (int i = j + 1; i < n; i++) {
            if (a[i] < a[iMin])
                iMin = i;
        }
        if (iMin != j) {
            int tempAJ = a[j];
            a[j] = a[iMin];
            a[iMin] = tempAJ;
        }
    }
}

File: Algorithms/Source Code Algorithms/Sorting/SelectionSort/SCSelectionSort.swift
func sort(_ a: inout [Int]) {
    for j in 0..<a.count {
        var iMin = j
        for i in j+1..<a.count {
            if a[i] < a[iMin] {
                iMin = i
            }
        }
        if (iMin != j) {
            let tempAJ = a[j]
            a[j] = a[iMin]
            a[iMin] = tempAJ
        }
    }
}
